aten/src/ATen/native/cuda/BinaryMiscOpsKernels.cu

```diff
@@ -139,7 +139,7 @@ void logical_xor_kernel_cuda(TensorIterator& iter) {
 void smooth_l1_kernel_cuda(TensorIterator& iter) {
   AT_DISPATCH_ALL_TYPES_AND(kHalf, iter.dtype(), "smooth_l1_cuda", [&]() {
     gpu_kernel(iter, [] GPU_LAMBDA(scalar_t a, scalar_t b) -> scalar_t {
-      auto z = fabs(a - b);
+      auto z = std::abs(a - b);
```

Comment:
OMG... With `fabs`, windows bulid is failing but after changing it to `std::abs`, ROCm fails...

Reply:
try ::abs